packages/devtools_app/lib/src/network/network_request_inspector.dart

```diff
@@ -36,73 +38,140 @@ class NetworkRequestInspector extends StatelessWidget {
 
   final NetworkRequest data;
 
+  @override
+  State<NetworkRequestInspector> createState() =>
+      _NetworkRequestInspectorState();
+}
+
+class _NetworkRequestInspectorState extends State<NetworkRequestInspector>
+    with TickerProviderStateMixin {
+  TabController _tabController;
+
+  List<DevToolsTab> _tabs;
+
+  int _currentTabControllerIndex = 0;
+
+  void _initTabController() {
+    _tabController?.removeListener(_onTabChanged);
+    _tabController?.dispose();
```

Comment:
I imagine we want to start tracking analytics on other cases we have tabs in DevTools. Would it make sense to define a AnalyticsTabController that handles logging the tab name each time a different tab is selected?
Seem annoying to have to change from a stateless to stateful widget just to make tabs work with our analytics.

Reply:
Done - Added AnalyticsTabbedView that handles sending tab switches to analytics.